packages/inter-protocol/src/stakeFactory/stakeFactory.js

```diff
@@ -190,16 +202,35 @@ export const start = async (
             'AdjustBalances',
           ),
         CloseVault: () =>
-          zcf.makeInvitation(seatx => helper.closeHook(seatx), 'CloseVault'),
+          zcf.makeInvitation(
+            seatx => helper.closeHook(seatx),
+            'CloseVault',
+            undefined,
+            StakeFactoryCloseProposalShape,
+          ),
```

Comment:
Seems annoyingly similar to the `zcf.makeInvitation` call in `stakeFactoryKit` function `makeCloseInvitation`. As a result, the proposalShape used here is annoyingly like the one used there. Can we consolidate both?

Leaving Conversation unresolved until I hear back.

Reply:
seems like they could be unified. What's the obstacle? The factory has the branded amounts. Can the kit get them from there? Is there an obstacle to the Kit making the branded amounts itself?

Reply:
@Chris-Hibbert I recall we discussed this, but I no longer remember where we landed. Do you?